### PR TITLE
Mfcc to database

### PR DESCRIPTION
**Pull request by ewega**
Sends MFCC features to database in its own table (we will then join on start time)
Also changed from 60 to 40 MFCCs. I think it might be more accurate for 4 seconds intervals. We can play around with the values anyways.

**HOW TO TEST**
The usual:
- cd audio-analysis
- make sure you have secrets.py in the directory
- turn on your localhost
- run python main.py log
- should work same old, slightly faster for longer videos

### CODE REVIEW

File: audio-analysis/lib/etl/sqlConnection.py [hunk 1]
```diff
@@ -60,17 +77,19 @@ def sqlConnectionSetup(engine):
...
+    elif table_name == 'mfcc_test_table':
+        mfcc_table = Table(table_name,meta,
+                    Column('mfcc_id',Integer, primary_key=True, autoincrement=True),
+                    Column('index',Integer),
+                    Column('end_time_s',Integer),
+                    Column('start_time_s',Integer),
+                    Column('url',String(length=500)),
+                )
+        for i in range (1,60):
+            engine.execute('ALTER TABLE `'+ table_name +'` ADD mfcc_'+ '{0:0=2d}'.format(i) +' float;')
```
Comment (tjlam) [inline audio-analysis/lib/etl/sqlConnection.py:89]: What does this do?

**Review by tjlam — APPROVED** [review]
LGTM 🐐 Just curious what the difference between `n_mfccs` = 40 vs 60

File: audio-analysis/lib/etl/sqlConnection.py (hunk 1 above)
Comment (ewega) [reply audio-analysis/lib/etl/sqlConnection.py:89]: This loops through the MFCC table and adds the appropriate columns (mfcc_1 to mfcc_40).

**Comment by ewega** [comment]
> LGTM 🐐 Just curious what the difference between `n_mfccs` = 40 vs 60

I was just seeing that most tutorials did 40 vs 60 for 4 second intervals. I think we will be able to play around with the value anyways, but wanted to take the standard and go forward with it

**Pull request merged**